store/src/java/com/zimbra/cs/mailbox/Mailbox.java

```diff
@@ -8156,6 +8162,38 @@ public void modifyContact(OperationContext octxt, int contactId, ParsedContact p
         return result;
     }
 
+    /**
+     * It modifies contact's first and full name with display name
+     * @param octxt
+     * @param listOfContactsToBeModified This list consist list of Map, InternetAddress as key and Pair of ContactId and 
```

Comment:
nit: remove a white space from the end of the string. Lines 8235, 8237, 8263 as well.

Reply:
Done